doc/how_to/custom_components/reactive_esm/examples/canvas_draw.md

```diff
@@ -0,0 +1,78 @@
+# Build a Custom Canvas Component
```

Comment:
I could not jslink as in the similar `ReactiveHTML` example.

```python
import panel as pn
import param

pn.extension()

class CanvasExample(pn.ReactiveESM):
        color = param.Color(default='#000000')
        line_width = param.Number(default=1, bounds=(0.1, 10))
        uri = param.String()

        _esm = """
        export function render({ data }) {
            console.log(data)
        }
        """

canvas = CanvasExample(height=400, width=400,styles={"background": "blue"})
png_view = pn.pane.HTML(height=400)

canvas.jslink(png_view, code={'uri': "target.text = `<img src='${source.uri}'></img>`"})
pn.Column(
    canvas.controls(['color', 'line_width'], jslink=True),
    canvas, png_view
).servable()
```

![image](https://github.com/holoviz/panel/assets/42288570/bbd90f1b-93d9-451b-8efb-02090cf659b6)

Reply:
Probably just needs an adjustment in `panel/links.py`. Will look.

Reply:
Fixed.